src/main/java/com/easypost/model/Shipment.java

```diff
@@ -1047,7 +1047,6 @@ public Rate lowestRate(final List<String> carriers, final List<String> services)
                 continue;
             }
             if (services != null && services.size() > 0 &&
-                    !services.contains(this.rates.get(i).getService().toLowerCase()) &&
```

Comment:
I thought we already removed this in separate PR?

Reply:
forgot to rebase, it's resolved.